### ISSUE TITLE
Can not upgrade with  timescaledb chart disabled

### ISSUE THREAD

**Issue by heigumjo** [timescale/tobs#120]
Hi we are running tobs with timescale outside kubernetes.
When I try to upgrade from tobs 0.2.1 with this command
```helm upgrade tobs timescale/tobs --namespace tobs -f custom-values.yaml ```
I get this error
```Error: UPGRADE FAILED: template: tobs/templates/NOTES.txt:158:94: executing "tobs/templates/NOTES.txt" at <{{template "timescaledb.fullname" $tsEnv}}>: template "timescaledb.fullname" not defined```

As I understand it the template timescaledb.fullname is defined in the timescaledb chart and hence undefined.

**Comment by VineethReddy02** [comment]
Hi @heigumjo 
what do you mean by outside Kubernetes? 

Also, can you share the exact steps you are following to upgrade your tobs setup?

**Comment by heigumjo** [comment]
We installed tobs using the tobs cli 0.2.1 with the following command:
`tobs install -f custom-values.yaml --namespace tobs`
When we upgraded we used tobs cli 0.3.0 and the command:
`helm upgrade tobs timescale/tobs --namespace tobs -f custom-values.yaml`

The timescale database is running external to kubernetes as we found that more practical.

Our custom values file look like this:
```yaml
#Custom values to use with helm to install monitoring with tobs
timescaledb-single:
  # disable the chart if an existing TimescaleDB instance is used
  enabled: false
promscale:
  enabled: true
   # connection options
  connection:
    # the db name in which the metrics will be stored
    dbName: &metricDB postgres
    # user to connect to TimescaleDB with
    user: postgres
    password:
      # Name of a secret object (templated) containing the connection password
      # Key must be value of promscale.connection.user
      # Created by default if timescaledb-single.enabled=true
      secretTemplate: &dbPassSecret timescaledb-passwords
    host:
      # Host name of the database instance
      nameTemplate: &dbHost nn.nnn.nnn.nnn
    port: nnnn
  resources:
    requests:
      memory: 4Gi
      cpu: 1
grafana:
  ingress:
    enabled: true
    path: /grafana/
    hosts:
      - www.someweb.com
## Grafana's primary configuration
## NOTE: values in map will be converted to ini format
  grafana.ini:
    server:
      root_url: https://www.someweb.com/grafana
      serve_from_sub_path: true
  timescale:
    database:
      enabled: true
      host:  *dbHost
      port: nnnn
      user: grafanadb
      pass: grafanadb
      dbName: *metricDB
      schema: grafanadb
      sslMode: require
    datasource:
      enabled: true
      user: grafana
      pass: grafana
      dbName: *metricDB
      sslMode: require
      host: *dbHost
      port: nnnn
    adminUser: postgres
    adminPassSecret: *dbPassSecret
prometheus:
  alertmanager:
    enabled: true
  server:
    timescaleRemote:
      remote_write:
        queue_config:
          capacity: 10000
          max_samples_per_send: 1000
          batch_send_deadline: 5s
  prometheus.yml:
    write_relabel_configs:
      - source_labels: [__name__]
        regex: 'go_.*'
        action: drop      
        remote_write:
          - url: http://tobs-promscale-connector.tobs.svc.cluster.local:9201/write
            queue_config: 
              max_shards: 30
              min_shards: 30
              capacity: 40000
              max_samples_per_send: 1000
              batch_send_deadline: 120s
```

**Comment by VineethReddy02** [comment]
I am curious to know why you are not using the tobs way of integrating to external timescaleDB mentioned [here](https://github.com/timescale/tobs/blob/master/docs/cli-usage.md#deploying-monitoring-stack)? Tobs natively supports integrating to external TimescaleDB. :) 
 
Also, Is there any reason you are upgrading directly using `helm upgrade tobs timescale/tobs --namespace tobs -f custom-values.yaml`?

We do have an `tobs upgrade` cmd to upgrade tobs instllation.

**Comment by heigumjo** [comment]
I tried the tobs command first, so I just sent the latest command that didn't work. (Sorry)

We have a history with tobs, mostly good, so I guess I haven't paid enough attention as things have changed. I'm pretty sure we followed recommend instructions at the time it was installed, but then again maybe not.

Since we have everything in an external database I think I'll just remove the existing installation and start fresh and install it using the tobs commands.
